code/modules/admin/verbs/beakerpanel.dm

```diff
@@ -65,8 +65,11 @@
 	set name = "Spawn reagent container"
 	if(!check_rights())
 		return
+/*
+	// Maybe some day, but this doesn't work without a client
 	var/datum/asset/asset_datum = get_asset_datum(/datum/asset/simple/namespaced/common)
 	asset_datum.send()
+*/
```

Comment:
```suggestion
	var/datum/asset/asset_datum = get_asset_datum(/datum/asset/simple/namespaced/common)
	asset_datum.send(usr.client||src)
```

Reply:
technically usr or src would be enough, but usr is lame and src could be the datum if this was manually called